sdk/appconfiguration/azure-data-appconfiguration/src/main/java/com/azure/data/appconfiguration/package-info.java

```diff
@@ -1,7 +1,30 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
+
 /**
- * Package containing classes for creating a {@link com.azure.data.appconfiguration.ConfigurationAsyncClient} to
- * perform operations on Azure App Configuration service.
+ * <p>The Azure App Configuration library perform operation on Azure App Configuration service.
+ * It helps developers centralize their application configurations simply and securely by storing
+ * all the settings for their application and secure accesses in one place.</p>
+ *
+ * <p>Azure App Configuration uses builder pattern to create an asynchronous or synchronous client.</p>
+ *
+ * <p><strong>Sample: Create a configuration client </strong></p>
+ *
+ * <p>See {@link com.azure.data.appconfiguration.ConfigurationClientBuilder} for how to use the client builder
+ * to create client before make any request to service.</p>
+ *
+ * <p>App Configuration supports two types of authentication methods.
+ * (1) Connection string, see {@link com.azure.data.appconfiguration.ConfigurationClientBuilder#connectionString(java.lang.String)}
+ * (2) Azure Active Directory, see {@link com.azure.data.appconfiguration.ConfigurationClientBuilder#credential(com.azure.core.credential.TokenCredential)}
+ * </p>
+ *
+ * <p>App Configuration support multiple operations, such as create, update, retrieve, and delete a configuration setting.
+ * See methods in client level class below to explore all capabilities that library provides.
+ *
+ * <p><strong>Sample: Use asynchronous App Configuration client </strong></p>
```

Comment:
Add detailed champion scenarios samples here for sync and async APIs via code snippet tool.
Describe what the samples are doing as well, like above.

Reply:
If we add champions scenarios in pakcage-info, does it repeating what the Client/AsyncClient  
JavaDoc already has?

Reply:
yeah repeating key champion scenarios here is fine.

Reply:
updated. @g2vinay